### PR TITLE
WIP: Update TB Variant Analysis tutorial

### PR DESCRIPTION
**Pull request by pvanheus**
* New version of tb_variant_filter
* More detail on filtering variants
* New version of tbprofiler

WIP because workflow not yet tested, because database that supports TB Variant Report is currently offline.

(apologies for use of main branch - a mistake)

### CODE REVIEW

File: topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md [hunk 1]
```diff
@@ -424,7 +447,7 @@ The next example is *SRR12416842* from an Indonesia [study](https://www.microbio
 >
 > 5. Run {% tool [samtools stats](toolshed.g2.bx.psu.edu/repos/devteam/samtools_stats/samtools_stats/2.0.3+galaxy0) %} {% icon tool %} on the *snippy on data XX, data XX, and data XX mapped reads (bam)* file. In the output, pay attention to the *sequences*, *reads mapped* and *reads unmapped* results.
 >
-> 6. Run the {% tool [BAM Coverage Plotter](toolshed.g2.bx.psu.edu/repos/iuc/jvarkit_wgscoverageplotter/jvarkit_wgscoverageplotter/20201223+galaxy0)} %} {% icon tool %} on the mapped reads BAM file that you got from **snippy**.
+> 6. Run the {% tool [BAM Coverage Plotter](toolshed.g2.bx.psu.edu/repos/iuc/jvarkit_wgscoverageplotter/jvarkit_wgscoverageplotter/20201223+galaxy0)} %} {% icon tool %} on the mapped reads BAM file that you got from **snippy** using the FASTA format reference you made with **seqret** as the reference.
```
Comment (gallardoalba) [inline topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md:450]:
```suggestion
> 6. Run the {% tool [BAM Coverage Plotter](toolshed.g2.bx.psu.edu/repos/iuc/jvarkit_wgscoverageplotter/jvarkit_wgscoverageplotter/20201223+galaxy0)%} {% icon tool %} on the mapped reads BAM file that you got from **snippy** using the FASTA format reference you made with **seqret** as the reference.
```

File: topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md [hunk 2]
```diff
@@ -308,7 +331,7 @@ Finally, TB Variant Report use the COMBAT-TB [eXplorer](https://explorer.sanbi.a
 We could go through all of the variants in the VCF files and read them out of a text table, but this is onerous and doesn't really give the context of the changes very well. It would be much nicer to have a visualisation of the SNPs and the other relevant data. In Galaxy we can use a tool called JBrowse.
 
 > <hands-on-title>Run JBrowse</hands-on-title>
->
+> 
 > 1. Use {% tool [seqret](toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS:%20seqret84/5.0.0) %} {% icon tool %} to convert the Genbank format reference (`Mycobacterium_tuberculosis_ancestral_reference.gbk`) to FASTA format. Use the following parameters:
```
Comment (gallardoalba) [inline topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md:335]:
```suggestion
> 1. Use **seqret** {% icon tool %} to convert the Genbank format reference (`Mycobacterium_tuberculosis_ancestral_reference.gbk`) to FASTA format. Use the following parameters:
```

**Comment by gallardoalba** [comment]
Those changes temporally fix the error; the problem is that spaces (%20) are not allowed in the tool link:

```toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS:%20seqret84/5.0.0```

But I'm not sure how to solve it; since you need the tutorial for Monday I guess that for now is enough to remove the broken link.

**Review by gallardoalba — DISMISSED** [review]

**Comment by hexylena** [comment]
Just documenting this in a more permanent/accessible place than matrix:

> Those changes temporally fix the error; the problem is that spaces (%20) are not allowed in the tool link:

They are. https://github.com/galaxyproject/training-material/pull/3282#discussion_r833064478
Unfortunately the linter I wrote is doing it's best to be helpful! where it specifically looks for a `%` that looks like you might've forgotten the closing `%}` of a liquid tag, and it's not smart enough (yet) to check for `%20` and exclude that from it's check, so it reports a warning that you maybe forgot to close this tag.

> for now is enough to remove the broken link.

I've restored the link, everything should work now. I've disabled the specific check ([GTN:006](https://github.com/galaxyproject/training-material/wiki/Error-Codes#GTN006)) for your tutorial.

**Review by hexylena — APPROVED** [review]

**Review by gallardoalba — APPROVED** [review]

**Pull request merged**